main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
```

Comment:
(edge cases) for inputs, consider adding restrictions or guard clause to what the user can enter as input so it will only accept strings